build.gradle

```diff
@@ -1,6 +1,7 @@
 allprojects {
 	apply plugin: 'java-gradle-plugin'
 	apply plugin: 'groovy'
+	apply plugin: "maven"
```

Comment:
I've reapplied the plugin, since `gradle install` is available with this, so the plugin can be tested locally